### ISSUE TITLE
Function model.train() only can be called once in your code. 尝试解决又有新问题

### ISSUE THREAD

**Issue by bbbxixixixi** [PaddlePaddle/PaddleX#649]
用pdx.seg.HRNet做语义分割，初步训练完成后做SWA，但是目前没有现成的，就自己循环模拟。但是model.train报了异常
[ERROR] Function model.train() only can be called once in your code.

在paddlex\cv\models\base.py中发现有这么一段
if hasattr(paddlex, 'model_built') and paddlex.model_built:
    logging.error(
        "Function model.train() only can be called once in your code.")
paddlex.model_built = True
所以在每个SWA的轮次开始，就将这个值置为False，并删除原来的model
  if model is not None:
      del model
      pdx.model_built = False

后来发现，在进行下一轮SWA训练并加载上一轮model参数后，发现模型加载完毕到出第一次结果的间隔时间越来越长
第一次约1分钟
2021-03-28 14:14:35 [INFO]	Load pretrain weights from Models/HRNet64/epoch_3.
2021-03-28 14:14:38 [INFO]	There are 1531 varaibles in Models/HRNet64/epoch_3 are loaded.
2021-03-28 14:15:40 [INFO]	[TRAIN] Epoch 1 finished, loss=1.537946, lr=0.001959 .
2021-03-28 14:15:58 [INFO]	Model saved in Models/HRNet64/swa1/epoch_1.

第二次约3分钟
2021-03-28 14:17:31 [INFO]	Load pretrain weights from Models/HRNet64/swa1/epoch_1/.
2021-03-28 14:17:35 [INFO]	There are 1531 varaibles in Models/HRNet64/swa1/epoch_1/ are loaded.
2021-03-28 14:20:57 [INFO]	[TRAIN] Epoch 1 finished, loss=1.282744, lr=0.001959 .
2021-03-28 14:21:23 [INFO]	Model saved in Models/HRNet64/swa2/epoch_1.

第三次约8分钟
2021-03-28 14:23:29 [INFO]	Load pretrain weights from Models/HRNet64/swa2/epoch_1/.
2021-03-28 14:23:33 [INFO]	There are 1531 varaibles in Models/HRNet64/swa2/epoch_1/ are loaded.
2021-03-28 14:31:09 [INFO]	[TRAIN] Epoch 1 finished, loss=0.982999, lr=0.001959 .
2021-03-28 14:31:44 [INFO]	Model saved in Models/HRNet64/swa3/epoch_1.

第四次约15分钟
2021-03-28 14:34:26 [INFO]	Load pretrain weights from Models/HRNet64/swa3/epoch_1/.
2021-03-28 14:34:30 [INFO]	There are 1531 varaibles in Models/HRNet64/swa3/epoch_1/ are loaded.
2021-03-28 14:49:06 [INFO]	[TRAIN] Epoch 1 finished, loss=0.891336, lr=0.001959 .
2021-03-28 14:49:49 [INFO]	Model saved in Models/HRNet64/swa4/epoch_1.

后续暂时没有尝试多少时间，这个有什么解决方法吗？清缓存？由于需要一次成型，所以重启不是个选项

**Comment by jiangjiajun** [comment]
这里指的`SWA`是？

**Comment by bbbxixixixi** [comment]
> 这里指的`SWA`是？
Stochastic Weights Averaging，周期式余弦退火额外再训练模型12个epoch，然后平均每个epoch训练得到的weights作为最终的模型
